index.php

```diff
@@ -1,477 +1,102 @@
 <?php
-/*
-File: 		index.php
-Coder:		Levi Meahan
-Created:	02/21/2012
-Revised:	11/25/2013 by Levi Meahan
-Purpose:	Authenticate/deauthenticate users, and direct logged in users to pages. Central gateway for entire game.
-Algorithm:	See master_plan.html
-*/
-
-//Start the session
+// Begin session & page load time
 session_start();
+$PAGE_LOAD_START = microtime(as_float: true);
 
-// Turn errors off unless Lsm
-if(!isset($_SESSION['user_id']) || $_SESSION['user_id'] != 1 || $_SESSION['user_id'] != 190) {
-	ini_set('display_errors', 'Off');
-}
-
-$PAGE_LOAD_START = microtime(true);
-
+// Load system
 require_once("classes/_autoload.php");
-$system = new System();
+$system = System::initialize();
 
+// Display errors on dev
 if($system->isDevEnvironment()) {
-    ini_set('display_errors', 'On');
+    ini_set(option: 'display_errors', value: 'On');
+}
+// Disable errors on production for non-developers
+else {
+    $value = 'Off';
+    if(isset($_SESSION['user_id']) && in_array($_SESSION['user_id'], System::$developers)) {
+        $value = 'On';
+    }
+    ini_set(option: 'display_errors', value: $value);
```

Comment:
It's okay to assume error display is off by default in prod (that's the way we want to set it for security reasons in config, not at runtime in this file)
```suggestion
    if(isset($_SESSION['user_id']) && in_array($_SESSION['user_id'], System::$developers)) {
        ini_set(option: 'display_errors', value: 'On');
    }
```

Reply:
Okay, changed that logic. Kept existing logic in place because I wasn't sure if it was needed.